### DISCUSSION TITLE
posts/2021-11-16_r-bioinfo-flow/

### DISCUSSION THREAD

**Discussion by giscus** [joelnitta/joelnitta-home#7] [Comments]
# posts/2021-11-16_r-bioinfo-flow/

How to combine Conda, Docker, and R to run modular, reproducible bioinformatics pipelines

https://www.joelnitta.com/posts/2021-11-16_r-bioinfo-flow/

**Comment by joelnitta** [comment]
Some follow-up... I just found out about a very handy feature of `bablewhale::run()`: the `debug` argument. Set this to `TRUE` to print out a docker command that you can run to enter the container. Very helpful to figure out what is wrong when your `bablewhale::run()` call doesn't go as expected. Thanks to @rcannood and @zouter for providing this great package!
